### ISSUE TITLE
Voronoi edges() and connected neighbors() convenience functions

### ISSUE THREAD

**Issue by nemo9955** [d3/d3-delaunay#97]
There should be functions for voronoi.neighbors() that returns the connected edge cells of the structure and voronoi.edges() for the edge cells .

I was shocked when I found this this observable after implementing a few hacks to prevent a messy graph, searched quite some time for a solution and found nothing. Maybe as an Issue other people will see the solution at least.

https://observablehq.com/@fil/unconnected-delaunay-neighbors?collection=@fil/voronoi

**Comment by Fil** [comment]
Closing as duplicate of https://github.com/d3/d3-delaunay/issues/96

Is this something that should be mentioned in the documentation?

**Comment by nemo9955** [comment]
Yes, if simply put in documentation would have been great for me.
Maybe as a footer with related algorithms/examples.

**Comment by Fil** [comment]
voronoi.neighbors are now available.

As to the suggestion for voronoi.edges(), would this be a manner of getting a list of cells that are clipped out? Note that some cells might be clipped out entirely and are not rendered at all. I'm wondering about the use case, but a solution might be to compare delaunay.neighbors(i).length and voronoi.neighbors(i).length : if they are equal then we have a "central" cell, if different an "edge" cell, and if voronoi.neighbors(i).length === 0 we have an "invisible" cell.

**Comment by nemo9955** [comment]
Basically I wanted to draw cells as a "river" from top to bottom , but the graph jumped the cells at the edge as they had smaller cost. 
I adapted [this code](https://github.com/d3/d3-delaunay/blob/b6e832962e99455ddaa375b114f3cc23c415866a/src/voronoi.js#L24) to get a boolean check of an edge cell (it was true for not rendered ones) and increased the cost of the graph edge if it was connected to an edge cell ... a big hack.
After implementing the true neighbors, I did not need to increase cost or create cells outside the view, so the edge request is redundant for me.
Thank you!
